src/components/ManageUsers/manage-users.css

```
@@ -0,0 +1,70 @@
+.usersearch__label {
+  margin-left: 4px;
+  font-size: 20px;
+  color: var(--color-primary);
+  font-weight: 600;
+}
+
+.usersearch__container {
+  display: flex;
+  flex-direction: column;
+}
+
+.usersearch__search {
+  margin: 10px 0;
+  width: 100%;
+  max-width: 600px;
+}
+
+.usersearch__input {
+  height: 16px;
+  width: 100%;
+  max-width: 600px;
+  background-color: #242424;
+  color: #000000;
```

I think the font color should be white. 
```suggestion
  color: var(--color-text);
```